app/assets/stylesheets/application.scss

```
@@ -12,28 +12,33 @@ main {
   padding: 1rem;
 }
 
-.sticky-top {
-  background-color: #5D99FF;
-}
+.container-login {
+  @extend .container-fluid;
+  max-width: 576px;
+  padding: 2rem;
+
```

閉じ括弧が抜けています！コンフリクトの解消後はサーバーを起動してエラーが発生していないか確認するようにしましょう！

```suggestion
}
```